pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/codestyle/xml/UnnecessaryModifier.xml

```diff
@@ -673,6 +673,20 @@ enum Foo {
     private Foo(String s) {
         name = s;
     }
+}
+        ]]></code>
+    </test-code>
+    <test-code>
+        <description>Static Modifier on interface</description>
+        <expected-problems>1</expected-problems>
+        <expected-messages>
+            <message>Unnecessary modifier 'static' on interface 'Bar': types nested within an interface type are implicitly static</message>
```

Comment:
The explanation is now wrong: it should read something like `member interfaces are implicitly static`

Reply:
it actually applies to all nested types, so I dont' think this is wrong at all..

```
public interface Foo {
  public class Bar { } // static here would be redundant
}
```